packages/sdk/src/browser-wallets/xverse/index.ts

```diff
@@ -25,10 +49,64 @@ function isInstalled() {
  * @returns An array of WalletAddress objects.
  */
 async function getAddresses(
-  _network: BrowserWalletNetwork = "mainnet",
-  _readOnly?: boolean,
+  network: BrowserWalletNetwork = "mainnet",
+  readOnly?: boolean,
 ): Promise<WalletAddress[]> {
-  return [];
+  if (!isInstalled()) {
+    throw new BrowserWalletNotInstalledError("Xverse not installed");
+  }
+
+  if (readOnly) {
+    throw new OrditSDKError("Read only mode is not supported on Xverse");
+  }
+
+  const result: Array<{
+    publicKey: string;
+    address: string;
+    format: AddressFormat;
+  }> = [];
```

Comment:
can we create an interface explicitly for this and export. incase any other projects want to extend it 🤔

Reply:
seems like from the fn return type already explicitly defined, lets use that here instead

Reply:
Fixed!